src/pages/Chat/index.jsx

```diff
@@ -65,8 +65,6 @@ const Chat = () => {
         >
           <div>
             <Input
-              // @ts-ignore
-              bgColor="gray"
```

Comment:
I'm receiving this JSDoc error because bgColor is missed.
![imagen](https://user-images.githubusercontent.com/66505715/156258022-42c2a3f7-5f35-4ce4-a117-72e160e78e88.png)

I would suggest to use bgColor="white" instead of delete it, and left `// ts-ignore` flag

`
              // @ts-ignore
              bgColor="white"
`

Reply:
ready